boileroom/backend/apptainer.py

```
@@ -0,0 +1,724 @@
+        -------
+        Any
+            HTTP client proxy object with methods like embed() that serialize inputs,
+            POST to /embed, and deserialize outputs.
+
+        Raises
+        ------
+        RuntimeError
+            If the backend has not been started.
+        """
+        if self._client is None:
+            raise RuntimeError("Apptainer backend is not initialized. Call start() before use.")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)